scheduler/src/test/java/org/cloudfoundry/autoscaler/scheduler/health/SchedulerHealthEndpointTest.java

```diff
@@ -20,26 +21,29 @@
 
   @Autowired private TestRestTemplate restTemplate;
 
-  @LocalServerPort private int healthServerPort;
+  @Autowired private MetricsConfiguration metricsConfig;
 
   @Test
-  public void greetingShouldReturnDefaultMessage() throws Exception {
-    String result =
-        this.restTemplate.getForObject(
-            "http://localhost:" + healthServerPort + "/metrics", String.class);
-    assertThat(result.contains("jvm_info"));
-    assertThat(result.contains("jvm_buffer_pool_used_bytes"));
-    assertThat(result.contains("jvm_buffer_pool_capacity_bytes"));
-    assertThat(result.contains("jvm_buffer_pool_used_buffers"));
-    assertThat(result.contains("jvm_gc_collection_seconds_count"));
-    assertThat(result.contains("jvm_gc_collection_seconds_sum"));
-    assertThat(result.contains("jvm_classes_loaded"));
-    assertThat(result.contains("jvm_classes_loaded_total"));
-    assertThat(result.contains("jvm_classes_unloaded_total"));
-    assertThat(result.contains("jvm_threads"));
-    assertThat(result.contains("jvm_memory_bytes"));
-    assertThat(result.contains("jvm_memory_pool_bytes"));
-    assertThat(result.contains("autoscaler_scheduler_data_source"));
-    assertThat(result.contains("autoscaler_scheduler_policy_db_data_source"));
+  public void greetingShouldReturnDefaultMessage() {
+    ResponseEntity<String> response =
+        this.restTemplate.getForEntity(
+            "http://localhost:" + metricsConfig.getPort() + "/metrics", String.class);
+    assertThat(response.getStatusCode().value()).isEqualTo(200);
+    String result = response.toString();
+    assertThat(result)
```

Comment:
this test ever actually asserted anything .... 😆 ... the assertThat is the new version import which required assertThat(xx).contains(yy)